data/human/hails.txt

```diff
@@ -742,10 +779,177 @@ phrase "friendly civilian"
+		"What has two heads, six feet, two arms and one tail?"
+		"What is the monster under your bed scared of?"
+		""
+		"What's black and white and red all over?"
+		""
+		"What does 1+1=?"
+		"How did the pond get an A in English?"
+		"Why do snowmen have noses?"
+		"What does a brain and winter have in common?"
+		"What did the system of Kornephoros say after figuring out the meaning of life?"
+		"Why do most scientists go diving?"
+		"What did the system of Sabik say after hearing a conspiracy theory?"
+		"Why did the system of Rastaban want to be a ballerina?"
+		"Why do secret agents wish that there was an earthquake on Martini?"
+		"What do Phurad and an oyster have in common?"
+		"Why was the pirate warlord happy after taking Pherkad?"
+		"What's worse than snakes on a ship?"
+		"What's the most common name on Vinci?"
+		"What did they change Uranus's name to?"
+		"Why doesn't the system of Kursa need a telescope?"
+		"Why was the pirate warlord disappointed after taking the system of Kochab?"
+		"Why wasn't anybody concerned when the system of Kaus Australis lost one of its legs?"
+		"Where should you go for a horn of plenty?"
+		"What do the Quarg say to people doing annoying things?"
+		"What do you call a pointless atmosphere?"
+		"Why is the system of Graffias a bad assassin?"
+		"Why does Gomeisa always have a cold?"
+		"What music does Girtab like?"
+		"Why is medical equipment so important in Gamma Corvi?"
+		"Why is Fomalhaut so boring?"
+		"Why do captains go to Elnath?"
+		"Why is it difficult to navigate through Dubhe?"
+		"Why do engineers go to Diphda?"
+	word
+		" "
+	word
+		"It was nice gnawing you!"
+		"To get to the other side!"
+		"At the space bar!"
+		"Whatever the Syndicate tell him it is!"
+		"Because the grass is greener on the other side!"
+		"A person on a horse!"
+		"You!"
+		"The Navy License!"
+		"A newspaper!"
+		"Your ship!"
+		"A window!"
+		"Because it could reed!"
+		"Because it s-nose!"
+		"They both s-know!"
+		"That's pretty Deep."
+		"Because they love the Deep!"
+		"That's a Longjump."
+		"Because there was a little Dancer in him!"
+		"Because they want it shaken, not stirred!"
+		"They both have a Pearl in them!"
+		"Because it gave him Solace!"
+		"Serpens 'round a system!"
+		"Leonardo!"
+		"Urass!"
+		"Because it's a Farseer!"
+		"Because it was only a Mere victory!"
+		"Because it was already a Hopper!"
+		"Kappa Centauri!"
+		"That's Enif of that!"
+		"Skymoot!"
+		"Because he can only Poisonwood!"
+		"Because he always s-Nifel-s!"
+		"Pieces with Harmony!"
+		"Lots of people are Bourne there!"
+		"Because it's run of the Millrace!"
+		"Because it has a good Pilot!"
+		"There's a big Haze!"
+		"Because they like to Tinker!"
+	word
+		""
+		" What? Not funny?"
+		" Hey, I thought it was good!"
+		" C'mon, at least laugh a little!"
+		" Why are you booing?"
+		" What's with the groaning?"
+		" Pretty clever, right?"
+		" What do you mean, two random phrases stuck together don't count?"
```

Comment:
I expect that most of the regular players (that don't read the datafiles) won't get the joke.